README.md

```diff
@@ -18,6 +18,11 @@ $ cd myApp
 $ make install-deps
 ```
 
+Create a local copy of the env file:
+```bash
+$ cp .env.dist .env
```

Comment:
Je suis pas super d'accord avec ce `.dist`. React de base ne versionne pas les `.env.*.local`, ce sont eux qui devrait override le `.env`. Le `.env` devrait être fournie par défaut pour toute application. Le truc qui est cool avec cette technique c'est que lors de l'installation pas besoin de ce `cp` :), ça marche "out of the box"